File: 1040 - Média 3.c
#include <stdio.h>

int main(void) {

  double nota1, nota2, nota3, nota4, media;

  scanf(" %lf %lf %lf %lf", &nota1, &nota2, &nota3, &nota4);

  media = ((nota1*2) + (nota2*3) + (nota3*4) + (nota4*1)) / 10;

  //testa pra ver se o aluno deve ir a exame

  printf("Media: %.1f\n", media);

  if (media < 5){

    printf("Aluno reprovado.\n");

  }else if (media >= 7){

    printf("Aluno aprovado.\n");

  }else{

    double exame;

    printf("Aluno em exame.\n");
    scanf(" %lf", &exame);
    printf("Nota do exame: %.1f\n", exame);
    

    media = (media + exame) / 2;

    media >= 5 ? printf("Aluno aprovado.\n") : printf("Aluno reprovado.\n");
    printf("Media final: %.1f\n", media);

  }

  return 0;
}

File: Python/Lista 3/1040 - Média 3.py
temp = input()
temp = temp.split()

n1 = float(temp[0])
n2 = float(temp[1])
n3 = float(temp[2])
n4 = float(temp[3])

media = ((n1*2) + (n2*3) + (n3*4) + (n4)) / 10


print("Media: {:.1f}".format(media))

if media >= 7:
    print("Aluno aprovado.")

elif media < 5:
    print("Aluno reprovado.")

elif 5 <= media < 7:
    print("Aluno em exame.")
    n5 = float(input())

    media = (media+n5) / 2

    print("Nota do exame: {:.1f}".format(n5))
    if media < 5:
        print("Aluno reprovado.")

    elif media >= 5:
        print("Aluno aprovado.")

    print("Media final: {:.1f}".format(media))
